qtconsole/tests/test_00_console_widget.py

```diff
@@ -274,11 +277,10 @@ def test_erase_in_line(self):
             cursor.insertText('')
 
     def test_link_handling(self):
-        noKeys = QtCore.Qt
-        noButton = QtCore.Qt.MouseButton(0)
-        noButtons = QtCore.Qt.MouseButtons(0)
-        noModifiers = QtCore.Qt.KeyboardModifiers(0)
-        MouseMove = QtCore.QEvent.MouseMove
+        noButton = QtCore.Qt.MouseButton.NoButton if IS_QT6 else QtCore.Qt.MouseButton(0)
+        noButtons = QtCore.Qt.MouseButton.NoButton if IS_QT6 else QtCore.Qt.MouseButtons(0)
+        noModifiers = QtCore.Qt.KeyboardModifier.NoModifier if IS_QT6 else QtCore.Qt.KeyboardModifiers(0)
+        MouseMove = QtCore.QEvent.Type.MouseMove if IS_QT6 else QtCore.QEvent.MouseMove
```

Comment:
The same about this enum? What if we add it to QtPy instead?

@dalthviz?

Reply:
Checking this I was able to use the scoped and unscoped versions without issue with QtPy 2.0.1 with PyQt5/6 and PySide5/6:

```python
QtCore.Qt.MouseButton.NoButton -> QtCore.Qt.NoButton
QtCore.Qt.KeyboardModifier.NoModifier -> QtCore.Qt.NoModifier
QtCore.QEvent.Type.MouseMove -> QtCore.QEvent.MouseMove
```